src/index.html

```diff
@@ -17,6 +29,87 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="moyo_header">
+      <div class="moyo_header_logo">
+        <a
+          href="#"
+          class="menu_link"
+        >
+          <img
+            src="images/logo.png"
+            alt="Logo"
```

Comment:
The `alt` attribute should describe the image content. Instead of 'Logo', provide a more descriptive alternative text.